Mathlib/Analysis/Calculus/ContDiff/Defs.lean

```diff
@@ -96,15 +96,7 @@ derivative, differentiability, higher derivative, `C^n`, multilinear, Taylor ser
 
 noncomputable section
 
-open NNReal Topology Filter
-
-/-
-Porting note: These lines are not required in Mathlib4.
-attribute [local instance 1001]
-  NormedAddCommGroup.toAddCommGroup AddCommGroup.toAddCommMonoid
--/
-
-open Set Fin Filter Function
+open NNReal Topology Filter Set Fin Function
```

Comment:
With two adjacent `open` lines, my default guess is that one of these should have been `open scoped` at the time of port. Can you check which of these imports are fine as `open scoped`, and demote them accordingly? Thanks!